AnimatedComponents/App.tsx

```diff
@@ -86,7 +86,7 @@ const App = () => {
         failedIcon={<CrossIcon />}
         successIcon={<TickIcon />}
       />
-      <Slider type={SliderType.SingleValueSlider} sliderWidth={300} min={0} max={100} step={5} />
+      <Slider type={SliderType.SingleValueSlider} />
```

Comment:
show range slider as well

Reply:
and how we can access the value of slider. 

make onChange function which will get triggered on slideEnd of tips.